src/popup/pages/API/SignTransaction.vue

```diff
@@ -275,12 +275,11 @@ export default {
         this.$notify({
           group: "notify",
           type: "error",
-          text: err.message,
+          text: this.caption,
```

Comment:
Sanitize this to prevent injection? See Matthew's report

Reply:
Added lodash escape in new commit